client/asset/interface.go

```diff
@@ -42,6 +42,9 @@ type WalletDefinition struct {
 	// description for each option. This can be used to request config info from
 	// users e.g. via dynamically generated GUI forms.
 	ConfigOpts []*ConfigOption `json:"configopts"`
+	// Managed can be set to true to disable wallet re-configuration through the
+	// standard UI.
+	Managed bool `json:"managed"`
```

Comment:
Any objections?

Reply:
Can you expand on the comment and the function of this bool more please?  I don't see how you intend to use this based on the diff in this PR.

Reply:
I'll probably remove this until I get the details worked out, but the idea is that if we're providing an external wallet, we already have a user interface and a way to configure the wallet, and having a separate interface to configure the wallet would be somewhat confusing and difficult to program around, for example if I needed to know when the configuration is updated for some external accounting purposes.